server/server_test.go

```diff
@@ -1886,11 +1886,29 @@ func TestCheckFlushRequest(t *testing.T) {
 		wantErrCode    codes.Code
 		wantErrDetails *spb.FlushResponseError
 	}{{
+		desc: "network instance is not specified",
+		inServer: &Server{
+			curElecID: &spb.Uint128{High: 0, Low: 3},
+		},
+		inRequest: &spb.FlushRequest{
```

Comment:
Do we have a success case that sets network-instance all in the flush request?

Reply:
Okay I see there is a "election ID specified explicitly - valid" which uses singleNIWithElection. I think that's good enough.

We do seem to be missing a multiNIWithElection flush all case though, but we can do that in a separate PR.